src/qml/components/StorageSettings.qml

```diff
@@ -27,13 +27,5 @@ ColumnLayout {
         actionItem: ValueInput {
             description: qsTr("c://.../data")
         }
```

Comment:
5. The storageLocation.qml component allows selecting the location for data storage. So providing the same option in the StorageSettings during the onboarding process seems redundant.

Reply:
Yes as it should be, the first screen you reference is the easy path settings, the next is a detailed settings screen.

Reply:
@jarolrod

Thanks for providing the clarification. But I still have some points to discuss.

The **storage location** page provides two options: The **Default Directory** and the **Custom Directory**. And as per my understanding, clicking on the **Custom Directory** will open the file manager to allow the user to choose the directories location. So having the exact same option in the storage setting still seems redundant to me.

Also, the Storage Settings page deals with the size-related factors regarding storage, so having a storage location option seems out of place when we have a dedicated onboarding page for the storage location selection.

I think it is better to keep the functionality of each page mutually exclusive and avoid overlap in their usage.

These are my thoughts on this matter, but I agree with your reasoning [here](https://github.com/bitcoin-core/gui-qml/pull/158#issuecomment-1223512830), and removing UI-based redundancies is out of the scope of this PR, so we can further brainstorm on this with the designer as well.